pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -1026,13 +1036,32 @@ func (ctrl *PersistentVolumeController) isVolumeReleased(volume *api.PersistentV
 // (it will be re-used in future provisioner error cases).
 func (ctrl *PersistentVolumeController) doDeleteVolume(volume *api.PersistentVolume) error {
 	glog.V(4).Infof("doDeleteVolume [%s]", volume.Name)
-	// Find a plugin.
+	var err error
+
+	// Find a plugin. Try to find the same plugin that provisioned the volume
```

Comment:
So, we expect this to fail for older volumes that were provisioned by the alpha version of this feature, right?